lib/Models/BingMapsSearchProvider.ts

```diff
@@ -39,7 +38,7 @@ export default class BingMapsSearchProvider extends SearchProvider {
     if (this.url.length > 0 && this.url[this.url.length - 1] !== "/") {
       this.url += "/";
     }
-    this.key = BingMapsApi.getKey(options.key);
+    this.key = options.key;
```

Comment:
Just double checking, do we need to do this for the other search providers as well? I noticed that this change was made to them as well.

Reply:
So the ones that were specifically deprecated and removed as of v1.73 were just the `BingMapsApi` and `MapboxApi`. I only fixed these because `BingMapsApi` was causing errors. I'm not sure what happened to the rest 🤔 but they seem to "work" just fine